### PR TITLE
Update Core-UI to v0.19.2 + replace new tabs

### PR DESCRIPTION
**Pull request by theocerutti**
**Component**: Tabs

/!\ Core-UI has been updated to its latest version.
Font Size is now based on rem this is why text size is larger.

**Design**:

image

image

image

### CODE REVIEW

File: ui/src/containers/NodePageRSP.js [hunk 1]
```diff
@@ -128,132 +122,74 @@ const NodePageRSP = (props) => {
...
+            <TextBadge variant={'infoPrimary'}>{alertsNode.length}</TextBadge>
+          ) : null
+        }
+        data-cy="alerts_tab_node_page"
+      >
+        <NodePageAlertsTab alertsNode={alertsNode} />
+      </Tabs.Tab>
+      <Tabs.Tab
+        path={`${url}/metrics`}
+        label="Metrics"
```
Comment (ChengYanJin) [inline ui/src/containers/NodePageRSP.js]:
```suggestion
        label={intl.formatMessage({ id: 'metrics' })}
```

**Review by ChengYanJin — COMMENTED** [review]
The code looks good to me!

**Review by ChengYanJin — APPROVED** [review]
LGTM!

**Comment by theocerutti** [comment]
@bert-e approve

**Pull request merged**